pureconfig/src/main/scala/com/avast/server/toolkit/pureconfig/PureConfigModule.scala

```diff
@@ -0,0 +1,42 @@
+package com.avast.server.toolkit.pureconfig
+
+import cats.data.NonEmptyList
+import cats.effect.Sync
+import cats.syntax.either._
+import pureconfig.error.{ConfigReaderFailure, ConfigReaderFailures, ConvertFailure}
+import pureconfig.{ConfigReader, ConfigSource}
+
+import scala.language.higherKinds
+import scala.reflect.ClassTag
+
+/** Provides loading of configuration into case class via PureConfig. */
+object PureConfigModule {
+
+  /** Loads the case class `A` using Lightbend Config's standard behavior. */
+  def make[F[_]: Sync, A: ConfigReader]: F[Either[NonEmptyList[String], A]] = make(ConfigSource.default)
+
+  /** Loads the case class `A` using provided [[pureconfig.ConfigSource]]. */
+  def make[F[_]: Sync, A: ConfigReader](source: ConfigSource): F[Either[NonEmptyList[String], A]] = {
+    Sync[F].delay(source.load[A].leftMap(convertFailures))
+  }
+
+  /** Loads the case class `A` using Lightbend Config's standard behavior or throws an exception. */
+  def makeOrThrow[F[_]: Sync, A: ConfigReader: ClassTag]: F[A] = makeOrThrow(ConfigSource.default)
```

Comment:
I would prefer to call it `makeOrRaise` such that it's clear that the exception is wrapped in `F[_]`

Reply:
Alright, sounds good to me.